paper.md

```diff
@@ -0,0 +1,77 @@
+    orcid: 0000-0002-0680-7097
+    affiliation: 2
+affiliations:
+ - name: Center for Humanities Computing, Aarhus University, Denmark
+   index: 1
+ - name: Interactive Minds Center, Aarhus University, Denmark
+   index: 2
+ - name: Department of Linguistics, Cognitive Science, and Semiotics, Aarhus University, Denmark
+   index: 3
+date: 17 March 2025
+bibliography: paper.bib
+---
+
+# Summary
+
+Turftopic is a topic modelling library including a number of recent topic models that go beyond bag-of-words and can understand text in context, utilizing representations from transformers.
+The library focuses on ease-of-use, providing a unified, interface for a number of different modern topic models, and boasting both model-specific and model-agnostic interpretation and visualization utilities.
+The user is afforded great flexibility in model choice and customization, but the library comes with reasonable defaults, not to overwhelm first-time users with a plethora of choices.
+In addition, our library is capable of modeling topics over time, modeling topics in streams of texts, finding hierarchical themes, and multilingual usage.
+For ease of interpretation, users can utilize large language models to assign human-readable names to topics, and can model key phrases, lemmas or stems right out of the box.
+
+![An Overview of Turftopic's Functionality](assets/paper_banner.png)
+
+# Statement of need
+
+While a number of software packages have been developed for contextual topic modelling in recent years, including BERTopic [@bertopic_paper], Top2Vec [@top2vec], CTM [@ctm], these packages include implementations of one or two topic models, and most of the utilities they provide are model-specific. This has resulted in the unfortunate situation that practitioners need to switch between different topic modelling libraries if they intend to try different models for their use case, and adapt to their particularities in both interface and functionality.
+Some attempts have been made at creating unified packages for modern topic models, including STREAM [@stream] and TopMost [@topmost]. These packages, however have a focus on neural models and topic model evaluation, have abstract and highly specialized interfaces, and do not include all popular topic models.
+Additionally, we considered the interpretation utilities included in these libraries fairly limited.
+
+Turftopic unifies state-of-the-art contextual topic models under a superset of the scikit-learn [@scikit-learn] API, which users are likely already familiar with, and can be readily included in scikit-learn workflows and pipelines.
+We focused on making Turftopic first and foremost an easy-to-use library, that does not necessitate expert knowledge or excessive amounts of code to get started with, but gives great flexibility to power users.
+The library also includes three topic models, which to our knowledge only have implementations in Turftopic, these are: KeyNMF [@keynmf], S^3^ [@s3] and GMM.
```

Comment:
You could stress that these are sota topic models (which I believe is fair at least for keyNMF and s^3)

Reply:
```suggestion
The library also includes three topic models, which to our knowledge only have implementations in Turftopic, these are: KeyNMF [@keynmf], S^3 [@s3], and GMM.
```